Switch build failing (broken compiler options)

https://circleci.com/gh/diasurgical/devilutionX/11688
`aarch64-none-elf-gcc: fatal error: /opt/devkitpro/libnx/switch.specs: attempt to rename spec 'link' to already defined spec 'old_link'`

Removing `set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -specs=${DEVKITPRO}/libnx/switch.specs")` from `/opt/devkitpro/switch.cmake` appears to solve the first issue, but then we run in to linker issues.

```
/opt/devkitpro/devkitA64/lib/gcc/aarch64-none-elf/10.1.0/../../../../aarch64-none-elf/bin/ld: /opt/devkitpro/libnx/lib/libnx.a(switch_crt0.o): in function `_start':
/home/fincs/pacman-packages/libnx/src/libnx-3.3.0/nx/source/runtime/switch_crt0.s:5: multiple definition of `_start'; /opt/devkitpro/devkitA64/lib/gcc/aarch64-none-elf/10.1.0/../../../../aarch64-none-elf/lib/crt0.o:(.text+0x0): first defined here
/opt/devkitpro/devkitA64/lib/gcc/aarch64-none-elf/10.1.0/../../../../aarch64-none-elf/bin/ld: cannot find -lfreetype
```

Last known good build was:
https://circleci.com/gh/diasurgical/devilutionX/11627

--- a/CMake/switch/devkita64-libnx.cmake
+++ b/CMake/switch/devkita64-libnx.cmake
@@ -23,6 +23,13 @@ set(PORTLIBS_PATH ${DEVKITPRO}/portlibs)
 set(PORTLIBS ${PORTLIBS_PATH}/switch)
 set(CMAKE_FIND_ROOT_PATH ${DEVKITA64} ${LIBNX} ${PORTLIBS})
 
+# FIXME
+# Workarounds to fix various linker errors
+set(CMAKE_EXE_LINKER_FLAGS_INIT "-march=armv8-a+crc+crypto -mtune=cortex-a57 -mtp=soft -fPIE -ftls-model=local-exec -L${DEVKITPRO}/portlibs/switch/lib -L${DEVKITPRO}/libnx/lib -specs=${DEVKITPRO}/libnx/switch.specs")
+set(CMAKE_MODULE_LINKER_FLAGS_INIT "${CMAKE_EXE_LINKER_FLAGS_INIT}")
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS_INIT}")
+set(CMAKE_EXE_MODULE_FLAGS "${CMAKE_EXE_MODULE_FLAGS_INIT}")
+
 # Set absolute tool paths:
 set(TOOLCHAIN_PREFIX ${DEVKITA64}/bin/aarch64-none-elf-)
 if(WIN32)

